opennebula/resource_opennebula_virtual_machine_test.go

```diff
@@ -955,6 +1012,10 @@ var testAccVirtualMachineTemplateConfigNICIPUpdate = testNICVNetResources + `
 		customer = "test"
 	  }
 
+	  nic {
```

Comment:
it seems there is twice same test

Reply:
Here, there is two NICs, I updated one of them and l left the other unmodified.
Sometimes I try to have several elements in a TypeList, and I add unmodified parts, it's a part of the test.

By the way, I rebase the branch on top of master.